src/controllers/oauth_controller.ts

```diff
@@ -292,7 +315,10 @@ OAuthController.get('/allow', auth, async (request: express.Request, response: e
 OAuthController.post('/token', async (request: express.Request, response: express.Response) => {
   const requestData = request.body;
   if (requestData.grant_type === 'authorization_code') {
-    handleAccessTokenRequest(request, response);
+    let token = await handleAccessTokenRequest(request, response);
```

Comment:
No response is sent in the error case here.

Reply:
Thanks for pointing this out! I didn't realize `handleAccessTokenRequest` had cases where it doesn't send an error.

Reply:
Oh, never mind, it does. I'll document the invariant properly though